Meadowlark-js/backends/meadowlark-postgresql-backend/src/repository/SqlHelper.ts

```diff
@@ -45,20 +47,11 @@ export function documentOwnershipByIdSql(documentId: string): string {
  * @returns SQL query string to determine document existence
  */
 export function checkDocumentExistsSql(documentId: string): string {
```

Comment:
Consider dropping this, as `existenceIdsForDocument` could provide same functionality. Alternatively, rename to be more specific to distinguish from existence checking via existence table.